frontend/openchat-agent/src/utils/caching.ts

```diff
@@ -130,12 +130,18 @@ type MigrationFunction<T> = (
     transaction: IDBPTransaction<T, StoreNames<T>[], "versionchange">,
 ) => Promise<void>;
 
+async function clearChatsStore(
```

Comment:
should be clearChitStore

Reply:
Nope. The chit is stored in the chat store so the migration clears the _chat_ store.